releases/release-6.2.0.md

```diff
@@ -0,0 +1,450 @@
+    - 修复 `max_sample_size` 为 `0` 时 ANALYZE 可能导致 panic 的问题 [#11192](https://github.com/tikv/tikv/issues/11192) @[LykxSassinator](https://github.com/LykxSassinator)
+    - 修复了启用 Raft Engine 时未清理加密密钥的问题 [#12890](https://github.com/tikv/tikv/issues/12890) @[tabokie](https://github.com/tabokie)
+    - 修复了 `get_valid_int_prefix` 函数与 TiDB 不兼容的问题，例如 `FLOAT` 类型被错误地转换成 `INT` [#13045](https://github.com/tikv/tikv/issues/13045) @[guo-shaoge](https://github.com/guo-shaoge)
+    - 修复了新创建的 Region Commit Log Duration 较高导致 QPS 下降的问题 [#13077](https://github.com/tikv/tikv/issues/13077) @[Connor1996](https://github.com/Connor1996)
+    - 修复 PD Region heartbeat 连接异常中断后未重新连接的问题 [#12934](https://github.com/tikv/tikv/issues/12934) @[bufferflies](https://github.com/bufferflies)
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        修复了限速备份后，BR 没有及时清理速度限制的问题 [#31722](https://github.com/pingcap/tidb/issues/31722) @[MoCuishle28](https://github.com/MoCuishle28)
+
+## Contributors
+
+感谢来自 TiDB 社区的贡献者们：
+
+- [u5surf](https://github.com/u5surf) in [#36345](https://github.com/pingcap/tidb/pull/36345)
+- [e1ijah1](https://github.com/e1ijah1) in [#36067](https://github.com/pingcap/tidb/pull/36067), [#35008](https://github.com/pingcap/tidb/pull/35008), and [#36065](https://github.com/pingcap/tidb/pull/36065)
+- [PrajwalBorkar](https://github.com/PrajwalBorkar) in [#36048](https://github.com/pingcap/tidb/pull/36048)
+- [likzn](https://github.com/likzn) in [#36024](https://github.com/pingcap/tidb/pull/36024)
+- [rahulk789](https://github.com/rahulk789) in [#36021](https://github.com/pingcap/tidb/pull/36021)
+- [virusdefender](https://github.com/virusdefender) in [#35999](https://github.com/pingcap/tidb/pull/35999)
+- [joycse06](https://github.com/joycse06) in [#35995](https://github.com/pingcap/tidb/pull/35995)
+- [morgo](https://github.com/morgo) in [#35992](https://github.com/pingcap/tidb/pull/35992) and others
+- [ixuh12](https://github.com/ixuh12) in [#35760](https://github.com/pingcap/tidb/pull/35760)
+- [blacktear23](https://github.com/blacktear23) in [#35560](https://github.com/pingcap/tidb/pull/35560)
+- [johnhaxx7](https://github.com/johnhaxx7) in [#35202](https://github.com/pingcap/tidb/pull/35202)
+- [GoGim1](https://github.com/GoGim1) in [#35143](https://github.com/pingcap/tidb/pull/35143)
+- [renbaoshuo](https://github.com/renbaoshuo) in [#35124](https://github.com/pingcap/tidb/pull/35124)
+- [Zheaoli](https://github.com/Zheaoli) in [#35089](https://github.com/pingcap/tidb/pull/35089)
+- [fanrenhoo](https://github.com/fanrenhoo) in [#34751](https://github.com/pingcap/tidb/pull/34751) and [#34047](https://github.com/pingcap/tidb/pull/34047)
+- [Willendless](https://github.com/Willendless) in [#13031](https://github.com/tikv/tikv/pull/13031)
+- [#v01dstar](https://github.com/v01dstar) in [#12810](https://github.com/tikv/tikv/pull/12810), [#12657](https://github.com/tikv/tikv/pull/12657), and [#12638](https://github.com/tikv/tikv/pull/12638)
```

Comment:
```suggestion
```

Reply:
Fixed in [d7358a3](https://github.com/pingcap/docs-cn/pull/10843/commits/d7358a3a32980ead0b9697a89abd3e1d88c57710).